css/styles.css

```diff
@@ -0,0 +1,79 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h5 {
+    margin: .5em 0;
+    padding: 0;
+}
+
+.main_container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: start;
+    width: 80%;
+    margin: 0 auto;
+}
+
+.collection {
+    justify-content: center;
+}
+
+.heading .footer {
+    justify-content: start;
+}
+
+.collection_item {
+    color: #000000;
+    margin: 5px 5px;
+    background-color: lightgrey;
```

Comment:
The use of a named colour is inconsistent. I suggest using hash values everywhere.

Reply:
Changed that to its hash value.